_plugins/brooklyn_metadata.rb

```diff
@@ -44,7 +44,7 @@ def generate(site)
         }
       end
       
-      url_set["git"] = "https://github.com/apache/brooklyn/tree/#{ git_branch }"
+      url_set["git"] = "https://github.com/apache/brooklyn-server/tree/#{ git_branch }"
```

Comment:
this could be wrong for some classes -- we need a more sophisticated strategy to determine if it's `server` or `library` etc (but this is probably good enough for now)